client/actions/Modals/index.js

```diff
@@ -10,24 +10,22 @@ import {
  * @param {string} modalName The name of the modal to set.
  * @param {!Object} modalProps The props to pass through the modal being set.
  */
-export const setModal = (modalName, modalProps) => dispatch => (
+export const setModal = (modalName, modalProps) => dispatch =>
   dispatch({
     type: MODAL_SET_MODAL,
     payload: {
       modalName,
       modalProps
     }
-  })
-);
+  });
 
 /**
  * Closes a modal (by removing it from the stack).
  * If "force" is set to true, the modal stack will be cleared/emptied.
  * @param {boolean} force Whether or not to clear the stack completely.
  */
-export const closeModal = (force = false) => dispatch => (
+export const closeModal = (force = false) => dispatch =>
```

Comment:
Move the returning `dispatch` to the next line

Reply:
You betcha!